build.sbt

```diff
@@ -41,6 +41,7 @@ lazy val baseSettings = Seq(
     "-Xlint:deprecation"
   ),
   resolvers ++= Seq(
+    Resolver.JCenterRepository,
```

Comment:
I believe that since Kamon is not used, its exclusion rules here can be dropped too.

Reply:
yeah... this is sad but necessary.   although metronome doesn't use kamon..  the dependency comes with using Marathon.   It does not build without this addition.